frontend/src/metabase/entities/users/forms.js

```diff
@@ -9,22 +9,22 @@ export default {
       {
         name: "first_name",
         title: t`First name`,
-        placeholder: "Johnny",
+        placeholder: t`Johnny`,
         autoFocus: true,
         validate: validate.maxLength(100),
         normalize: firstName => firstName || null,
       },
       {
         name: "last_name",
         title: t`Last name`,
-        placeholder: "Appleseed",
+        placeholder: t`Appleseed`,
         validate: validate.maxLength(100),
         normalize: lastName => lastName || null,
       },
       {
         name: "email",
         title: t`Email`,
-        placeholder: "nicetoseeyou@email.com",
+        placeholder: t`nicetoseeyou@email.com`,
```

Comment:
Do we really need to translate an email address?

Reply:
+1 likely not @WiNloSt

Reply:
Fixed in https://github.com/metabase/metabase/pull/33123/commits/9d82e2a679ffa0f62e6b48421115caff3c8ccbb4